tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_IMAP_email_notification.py

```diff
@@ -64,6 +64,7 @@ def test_IMAP_email_notification(self):
         self.apps.switch_to_displayed_app()
 
         # check if the sender's email address is fine
+        self.wait_for_condition(lambda m: email.senders_email == mock_email.senders_email)
```

Comment:
The first call to `email.senders_email` returns "" (empty string). Therefore the next assertion will always fail. But the second call will pass. Let me see if I can make if more elegant.